Translate Phantom Thief Succubus(10070) (#475)
Giratena/Succubus-Duel-English-Translation: CSV/Language/English/Card/10070.txt

EU70_handJobAnime,
EU70_handJobWithKissAnime,
EU70_bustShakeAnime,

> 卓越した盗みの技術を持つサキュバス。人の認知を操り、認識されない状態であらゆる場所に忍び込むことができる。十分に警戒していたはずでも、気が付くと彼女に密着され濃厚なキスを浴びせられ、たいていの場合、そのまま骨抜きにされてしまう。彼女の攻撃を一切受けずに戦うことは困難であり、もし彼女に勝つことを望むなら、サキュバスの甘い快楽攻撃を受けてもそれを振り払える強い精神力が必要である。,A succubus with exceptional thieving skills able to manipulate people's perception and infiltrate any location without being detected. Even when you think you're being careful enough; she'll be pressed against you delivering deep passionate kisses before you know it; and—in most cases—you'll end up completely under her spell. Avoiding her attacks while fighting her often proves difficult; and if you want any hope of defeating her; you'll need strong mental fortitude to shake off this succubus's sweet pleasure attacks.

Reviewer: Hmm maybe this can be touched up a little for flow. There are some English rules here of why it need to be this way I'm sure but I don't know them well enough to explain properly.

Also side note, I think 甘い in this case, is reffering to 'skilled' rather than 'sweet'

```suggestion
卓越した盗みの技術を持つサキュバス。人の認知を操り、認識されない状態であらゆる場所に忍び込むことができる。十分に警戒していたはずでも、気が付くと彼女に密着され濃厚なキスを浴びせられ、たいていの場合、そのまま骨抜きにされてしまう。彼女の攻撃を一切受けずに戦うことは困難であり、もし彼女に勝つことを望むなら、サキュバスの甘い快楽攻撃を受けてもそれを振り払える強い精神力が必要である。,A succubus with exceptional thieving skills capable of manipulating people's perception and able to infiltrate any location without being detected. No matter how alert you believe yourself to be; before you know it she'll be pressed up against you—her lips against yours—making out with you in deep passionate kisses. In most cases you'll end up completely under her spell. Avoiding her attacks while fighting her often proves difficult; and if you want any hope of defeating her; you'll need strong mental fortitude in order to shake off her skilled pleasure attacks.
```

> \n　彼女は特にキスが得意で、快楽に流されまいと固く口を閉じて身を守っても、舌でいつの間にか唇の中に侵入され、心をからめとるような舌遣いで脳を一瞬で蕩けさせてくる。あまりの気持ちよさに唇が離れたあとも彼女の顔をうっとりと眺めることしかできなくなれば、あとは顔中にキスマークをまぶされ、心の深いところまで魅了されてしまうだろう。そうなれば最後、二度と魅了を解くことはできず、あとは彼女の所有物としてその後の一生を過ごすことになるのである。,She's particularly skilled at kissing; and no matter how tightly you close your lips to defend yourself against being swept away by pleasure; her tongue somehow slips past; ensnaring your heart with its skilled movements and melting your brain in an instant. Should the overwhelming pleasure leave you unable to do anything but dreamily gaze at her face once your lips part; she'll scatter kiss marks all over your face; charming you to the very depths of your soul. Once that happens; the charm can never be broken; and you'll spend the rest of your life as her plaything.

Reviewer: Hmm I feel like this description flows pretty poorly in English as well. Rewording the structure of these things is key. Here's a suggestion and let me know what you think:

```suggestion
\n　彼女は特にキスが得意で、快楽に流されまいと固く口を閉じて身を守っても、舌でいつの間にか唇の中に侵入され、心をからめとるような舌遣いで脳を一瞬で蕩けさせてくる。あまりの気持ちよさに唇が離れたあとも彼女の顔をうっとりと眺めることしかできなくなれば、あとは顔中にキスマークをまぶされ、心の深いところまで魅了されてしまうだろう。そうなれば最後、二度と魅了を解くことはできず、あとは彼女の所有物としてその後の一生を過ごすことになるのである。,She's particularly skilled at kissing; and though you may try to seal your lips shut to avoid being swept away by her pleasures; no matter how you defend yourself; her tongue will always somehow slip past; allowing her to instantly melt your mind with her seductive and heart-captivating techniques. Should it ever reach the point where you feel so good you're unable to do anything but gaze dreamily at her face whenever her lips part; she'll pepper you with kisses; leaving her lipstick all over your face and charming you to the very depths of your soul. Once that happens; you will never recover from being charmed by her; and you'll spend the rest of your life as her plaything.
```

[1].このカードは「隠密」「ステルス」を持つ。このカードは相手プレイヤーに直接攻撃できる。,[1]. This card has "Stealth" and "Concealment." This card can attack the opposing player directly.

> [2].このカードが相手プレイヤーに戦闘ダメージを与えたとき発動する。与えたダメージの分だけ自分プレイヤーのLPを回復する。,[2]. When this card deals combat damage; it recovers LP equal to the damage dealt.

Reviewer: Missing the condition she needs to attack directly to trigger.

```suggestion
[2].このカードが相手プレイヤーに戦闘ダメージを与えたとき発動する。与えたダメージの分だけ自分プレイヤーのLPを回復する。,[2]. Whenever this card deals combat damage to an opponent player; recover LP equal to the damage dealt.
```

> (ハード以上の場合、さらにデッキからトラップカードをランダムに１枚手札に加える),(On Hard difficulty; this effect also adds 1 random trap card from your deck to your hand)

Reviewer: ```suggestion
(ハード以上の場合、さらにデッキからトラップカードをランダムに１枚手札に加える),(On Hard difficulty; when this effect is triggered; also add 1 random trap card from your deck to your hand)
```

Author: 'Additionally add' feels really weird. Maybe just 'it also adds'?

[3].このカードが破壊され墓地へ送られたとき発動する。「怪盗の挑戦状」を1枚生成し手札に加える。,[3]. When this card is destroyed and sent to the graveyard; create 1 "Phantom Thief's Challenge" and add it to your hand.

> あなたの大事なエナジー、すぐにでも盗んであげる❤\nほら、グローブを亀頭にかぶせて・・・❤くる、くる❤シコ、シコ❤\n扱かれながら先っぽをこねこねされるの、たまらないでしょ？,I'll steal your precious energy in a flash~❤ Look; I'll start by slipping my glove over the head of your cock...❤ Twist; twist~❤ Stroke; stroke~❤ Getting jerked off while your tip is kneaded... You can't resist this pleasure; can you?

Reviewer: A suggestion I think might flow better:
```suggestion
あなたの大事なエナジー、すぐにでも盗んであげる❤\nほら、グローブを亀頭にかぶせて・・・❤くる、くる❤シコ、シコ❤\n扱かれながら先っぽをこねこねされるの、たまらないでしょ？,I'll steal your precious energy in a flash~❤ Look; I'll start by slipping my glove over the head of your cock...❤ Twist; twist~❤ Stroke; stroke~❤ It's hard to hold it in when I pump your dick while kneading the tip at the same time; isn't it?
```

> あは、いい表情❤その反応、大好きよ❤\nかっこいい顔をしてた男の子が、手の平をくるくる回されただけで、\n喘いでビクビクすることしかできなくなるの…❤楽しいと思わない？,Aha; what a nice expression.❤ I love that reaction~❤ A handsome boy can put on a cool face all he wants; but as I swirl my palm around him like this; all he can do is moan and twitch...❤ Rather pleasant; don't you think?

Reviewer: 楽しい means 'fun'/'enjoyable' not always implying pleasant.

```suggestion
あは、いい表情❤その反応、大好きよ❤\nかっこいい顔をしてた男の子が、手の平をくるくる回されただけで、\n喘いでビクビクすることしかできなくなるの…❤楽しいと思わない？,Aha; it looks like you're really getting into it.❤ I love little reactions like that~❤ A handsome boy can put on a cool face all he wants; but as soon as I rub my palms all over his dick like this; all he can do is moan and twitch...❤ It really is fun; don't you think?
```

Author: See I originally had it that way, but describing this as 'fun' in any way didn't feel right.

Reviewer: Hmm. The nuance here is that the idea of teasing is fun to her. It should also be something that ties in with the next sentence as well. 
"this is fun; isn't it?"
"though I suppose on the receiving end; "fun" is a bit of a lighthearted way of putting it; don't you think?"

> ふふ、される側からしたら「楽しい」なんて軽いものじゃ済まない\nかしら。あへあへ声をあげて、腰をビクンビクンって跳ねさせて❤\nもうイきたい？イっていいわよ❤ほら、シコ、シコ❤,Fufu; though I suppose on the receiving end; "pleasant" doesn't really cover just how intense it feels; does it? Making those pathetic sounds; your hips jerking and bouncing like that...❤ Do you want to cum already? Then go on; do it.❤ Come on; stroke; stroke~❤

Reviewer: ```suggestion
ふふ、される側からしたら「楽しい」なんて軽いものじゃ済まない\nかしら。あへあへ声をあげて、腰をビクンビクンって跳ねさせて❤\nもうイきたい？イっていいわよ❤ほら、シコ、シコ❤,Fufu; though I suppose on the receiving end; "fun" is a bit of a lighthearted way of putting it; don't you think? Making those pathetic sounds; your hips jerking and bouncing like that...❤ Do you want to cum already? Then go on; do it.❤ Come on; stroke; stroke~❤
```

Reviewer: Updated suggestion to match the previous comment of "fun"

あら、踏まれて喜んでる。\nあなたマゾなのね❤,Oh my; you're actually enjoying being stepped on! You're quite the masochist; aren't you?❤

> マゾは簡単でいいわ❤\n優しい言葉をかけなくても、心の防御が最初から隙だらけ、\nちょっといじめてあげればすぐ虜になるんだもの❤,Masochists are so easy to handle~❤ I don't even need to pretend to be nice to them. Their mental defenses are so full of holes from the start; I just need to tease them a little and they become completely captivated right away.❤

Reviewer: ```suggestion
マゾは簡単でいいわ❤\n優しい言葉をかけなくても、心の防御が最初から隙だらけ、\nちょっといじめてあげればすぐ虜になるんだもの❤,Masochists are so easy to handle~❤ I don't even need to pretend to be nice to them. Their mental defenses are so full of holes from the start; so I just need to tease them a little and they become completely captivated right away.❤
```

Author: I don't feel like 'so' adds anything here, especially since it's said right before where you added it. Cause and effect sentences can exist just fine without it

> ね❤あなたもこうやって足でぐりぐり踏まれて、\n心の扉をこじ開けられたいんでしょ？\n無理やり虜にして、わたしにエナジー貢ぐだけの存在にしてあげる❤,Isn't that right?❤ You want to be stepped on and ground under my foot too; don't you? To have the doors to your heart pried open by me. I'll force you into captivation and make you into someone who only exists to offer me energy~❤

Reviewer: That first sentence is a little bit weird. the phrase "do you want to" should be linked to wanting the doors to the heart opened, NOT being stepped on. Make sure the verbs are linked to the correct sentence. Something like this should flow better, and will be much closer to what she's actually saying:

```suggestion
ね❤あなたもこうやって足でぐりぐり踏まれて、\n心の扉をこじ開けられたいんでしょ？\n無理やり虜にして、わたしにエナジー貢ぐだけの存在にしてあげる❤,Hey~❤ You want to me rub your penis under my foot just like this; and open the gates to your heart as well; don't you? I'll force you into captivation and make you into someone who only exists to offer me energy~❤
```

Author: The way I wrote it starches attached it to both. All those suggestions feel really weird and jarring by uncoupling then like that. What if I emdashed it?

Isn't that right?❤ You want to be stepped on and ground under my foot—To have the doors to your heart pried open by me—too; don't you? 

Or something. They're fairly dependent and connected ideas because they're in effect the same action, so trying to place hard separation on them makes it feel weird

Reviewer: Hmm that suggestion doesn't sound right either. What's wrong with this?
> By letting me rub your cock under my foot like this... You want to me to use my techniques to open the gates to your heart; don't you?

I think they should be linked because him letting her results in her comment suggestion of opening the gates to his heart. It also flows better too I think.

Author: The 'by letting me' if the part that feels off. I feel like the line isn't "If I do A, then B." It feels more like "You want A and, by extension, B.' She's confirming he's a masochist and that he wants both he things. That's the thing, they SHOULD be connected, and your line is disconnecting them.

Reviewer: Hm isn't it more "if A happens, therefore B must be true"? What about this then?

> If you're letting me rub your cock under my foot like this... That means you want me to use my techniques to open the gates to your heart; don't you?

But this could be already implied. Not sure.

Author: Every time I look into the translation, it isn't a cause-effect. It's just two different "you want me to"s The fact that she's stepping on him doesn't make him open his heart. Both are just side effects of him being a masochist. It's like "Because you're a masochist, you want me to step on you. Because you're a masochist, you want me to pry the gates of your heart open."

Reviewer: Discussed on discord. Updated suggestion!

> この服を着ていると、\nへそを見ちゃう人とおっぱいを見ちゃう人に分かれるの。\nふふ、あなたはおっぱいばかり見ているわね❤,When I wear this outfit; people seem to get split between two types—those who look at my navel; and those who look at my breasts. Fufu; you're only looking at my breasts; aren't you?❤

Reviewer: Maybe stare fits the -ちゃう ender better.
Also 'midriff' is a much more common word for this context. You also don't need to repeat the last sentence topic again in English. It's already covered:
```suggestion
この服を着ていると、\nへそを見ちゃう人とおっぱいを見ちゃう人に分かれるの。\nふふ、あなたはおっぱいばかり見ているわね❤,When I wear this outfit; people seem to get split between two types—those who stare at my midriff; and those who stare at my breasts. Fufu; you're certainly the latter; aren't you?❤
```

Author: I think it's okay, but I think we should drop the 'and'

> 味方の守りをかいぐぐられて、自分自身がピンチだというのに、\nずいぶん呑気❤マスターくんはおっぱいに夢中で\nそんなことどうでもいいのかしら？くすくす❤,Even though your allies' defenses have been breached and you're in grave danger yourself; you still seem so carefree.❤ Are you so absorbed in my breasts that such things don't matter to you? *Giggle*❤

Reviewer: 'still' should be OK to remove here ^^
かしら means 'perhaps' and implies rumination

```suggestion
味方の守りをかいぐぐられて、自分自身がピンチだというのに、\nずいぶん呑気❤マスターくんはおっぱいに夢中で\nそんなことどうでもいいのかしら？くすくす❤,Even though your allies' defenses have been breached and you're in grave danger yourself; you seem rather untroubled~❤ Perhaps you're so focused on my breasts that such things no longer concern you? *Giggle*❤
```

Author: Your last sentence is no longer a question. We'd need to either change it to a period or change the beginning to something like 'Are you perhaps'

Reviewer: https://www.reddit.com/r/grammar/comments/e4msy/should_sentences_that_typically_begin_with_maybe/
Either are possible. Especially in dialogue to keep the tone. You can keep the question mark or remove it, but I think it flows better (and is more accurate) with the question mark.

> 女のおっぱいがそんなに好きなら\nほら、服の下からあなたのチンポ入れて、\nたっぷり挟んであげる❤,If you love women's breasts that much; here; I'll slip your cock under my clothes and squeeze it tight between them~❤

Reviewer: ```suggestion
女のおっぱいがそんなに好きなら\nほら、服の下からあなたのチンポ入れて、\nたっぷり挟んであげる❤,If you love women's breasts that much; here. I'll slip your cock under my clothes and squeeze it tightly between them~❤
```

> 女の子のおっぱいが好きなの？\nそれなら、ほら、服の下からあなたのチンポ入れて、\nたっぷり挟んであげる❤,Do you like girls' breasts? Then; here; I'll slip your cock under my clothes and squeeze it tight between them~❤

Reviewer: ```suggestion
女の子のおっぱいが好きなの？\nそれなら、ほら、服の下からあなたのチンポ入れて、\nたっぷり挟んであげる❤,Do you like girls' breasts? Then here. I'll slip your cock under my clothes and squeeze it tightly between them~❤
```

> 服でぎゅっとしめつけられて、チンポ幸せでしょう？\nここは甘い快楽の牢屋❤チンポを捕まえられたらもう逃げられない❤\nたぱんたぱんって優しく揺らすだけで、頭の中まっしろ❤,Is your cock happy to be trapped in my clothes? It's like a sweet prison of pleasure.❤ Once your penis is caught like this; you have no hope of escape.❤ Just gently bouncing them makes your head go completely blank~❤

Reviewer: I think it helps flow to include breasts here. Also 頭の中まっしろ uses the しろ ending, which is like giving a command, so something like this should be more accurate too! Try something like this:

```suggestion
服でぎゅっとしめつけられて、チンポ幸せでしょう？\nここは甘い快楽の牢屋❤チンポを捕まえられたらもう逃げられない❤\nたぱんたぱんって優しく揺らすだけで、頭の中まっしろ❤,Is your cock happy to be trapped under my clothes? It's like a sweet prison of pleasure.❤ Once your penis is caught like this; you have no hope of escape.❤ All I have to do is gently bounce my breasts like this; and your head will go completely blank~❤
```

> ふふ、あなたの方が怪盗に捕まえられるなんてあべこべね❤\nあなたの心もエナジーも、全部わたしのおっぱいで奪ってあげる❤,Fufu; a virtuous man being captured by a phantom thief... Rather ironic; don't you think?❤ I'll steal everything from you with my breasts—both your heart and your energy~❤

Reviewer: ```suggestion
ふふ、あなたの方が怪盗に捕まえられるなんてあべこべね❤\nあなたの心もエナジーも、全部わたしのおっぱいで奪ってあげる❤,Fufu; a virtuous man; captured by a phantom thief... Rather ironic; don't you think?❤ I'll use my breasts to steal everything from you—your heart and your energy~❤
```

> どれだけ守りのカードを並べても無駄よ？\nわたしの動きは変幻自在。敵の守りを抜けることなんてたやすいわ。\nこうやって一気に接近して・・・❤,Haven't you realized? It doesn't matter how many defensive cards you line up. My movements are ever-changing. Slipping past enemy defenses is child's play to me . It's always so easy to close in all at once. Like so...❤

Reviewer: Flow change. Also 接近して has more intimate connotations which we should try to preserve!

```suggestion
どれだけ守りのカードを並べても無駄よ？\nわたしの動きは変幻自在。敵の守りを抜けることなんてたやすいわ。\nこうやって一気に接近して・・・❤,Don't you see? It doesn't matter how many defensive cards you line up. My movements are ever-changing. Slipping past enemy defenses is child's play to me. Before you know it; we'll be pressed up against each other like so...❤
```

> 相手マスターを胸に埋めてしまうことだって簡単ね❤\nむに、むに❤ぱふぱふ❤\nお姉さんのおっぱいは気持ちいい？くすくす❤,Moving like this; burying the opposing master's face in my chest is such a simple task.❤ Squish; squish~❤ Puff-puff~❤ Don't my breasts feel amazing? *Giggle*❤

Reviewer: Hmm you might not even need "moving like this" here! It's not in the Japanese.
```suggestion
相手マスターを胸に埋めてしまうことだって簡単ね❤\nむに、むに❤ぱふぱふ❤\nお姉さんのおっぱいは気持ちいい？くすくす❤,Burying the opposing master's face in my chest is such a simple task.❤ Squish; squish~❤ Puff-puff~❤ Don't my breasts feel amazing? *Giggle*❤
```

Author: I know, it was more a localization choice to connect the action to the result in a more natural way

Reviewer: I think it actually flows better without so we should be able to remove it.

> 仲間のことなんて忘れていいのよ❤\nやわらかぁい胸の感触におぼれて、ただおっぱいに甘えればいいの❤\n分かったかしら、マスターのぼうや❤,Go ahead and just forget all about your allies!❤ Just drown in the soft sensation of my chest and let me pamper you with my breasts.❤ Doesn't that sound nice; master boy?❤

Reviewer: いいのよ here is more implying she can say "you can do it, no one will mind" rather than an exclamation point. Master boy also sounds a little weird, but maybe it's ok?
```suggestion
仲間のことなんて忘れていいのよ❤\nやわらかぁい胸の感触におぼれて、ただおっぱいに甘えればいいの❤\n分かったかしら、マスターのぼうや❤,It's okay; forget all about your allies~❤ Just sink into the embrace of my soft breasts; and let me pamper you properly.❤ Doesn't that sound nice; master boy?❤
```

Author: I dont like 'cardmaster boy.' It's too much of a mouthful to be the biting, condescending nickname is supposed to be.

Reviewer: Alright, feel free to leave the boy as is, but the rest should be kept.

> 相手の視線を誘導するのは盗みの基本よね。\nおっぱいを両手で持ち上げて、ゆさってすれば、\nほら、あなたも視線が釘付け❤,Redirecting your opponent's gaze is pretty basic thievery. If I lift my breasts with both hands and shake them like this... See? I can control your gaze; too~❤

Reviewer: ```suggestion
相手の視線を誘導するのは盗みの基本よね。\nおっぱいを両手で持ち上げて、ゆさってすれば、\nほら、あなたも視線が釘付け❤,Being able to redirect your opponent’s focus is a basic part of thievery. If I lift my breasts with both hands and shake them like this... See? I can divert your attention, too~❤
```

Author: I like this except for your choice to double up on 'redirect'

Reviewer: Maybe 'distract your focus'?

Author: I don't think 'distract your focus' is a thing.

Reviewer: What about: "divert your attention"/"disrupt your concentration"/"distract you"?

If you don't like those ones, happy to go with one that you think is better.

Author: I think divert your attention could work

> この隙に持ち物を盗むのが常套手段だけど、\nわたしが今欲しいのはあなたのエナジーだし、そうね、\nおっぱい揺らしながらチンポをいじめてあげればいいのかしら。,Taking advantage of this opening to steal belongings is pretty standard practice; but what I want right now is your energy; so I suppose I should tease your cock while bouncing my breasts instead.

Reviewer: ```suggestion
この隙に持ち物を盗むのが常套手段だけど、\nわたしが今欲しいのはあなたのエナジーだし、そうね、\nおっぱい揺らしながらチンポをいじめてあげればいいのかしら。,Taking advantage of this opening to steal belongings is pretty standard practice; but what I want right now is your energy; so perhaps I should tease your cock while bouncing my breasts instead.
```

> 顔の前でわざとらしく胸を揺らして…❤頭バカになっている間に、\n膝でぐりぐり❤ぐりぐり❤\nふふ、効果てきめんみたい❤とってもアホ面よ、あなた。くすくす❤,First I openly shake my chest in front of your face...❤ While your head goes stupid; I'll grind my knee against your crotch.❤ Grind; grind~❤ Fufu; looks like it's extra effective.❤ You look completely idiotic. *Giggle*❤

Reviewer: ```suggestion
顔の前でわざとらしく胸を揺らして…❤頭バカになっている間に、\n膝でぐりぐり❤ぐりぐり❤\nふふ、効果てきめんみたい❤とってもアホ面よ、あなた。くすくす❤,First; I'll make a display of shaking my chest in front of your face...❤ Then; as your head goes stupid; I'll grind my knee against your crotch.❤ Grind; grind~❤ Fufu; looks like it's extra effective.❤ You look completely idiotic. *Giggle*❤
```

んっちゅ…❤キスをしながら、扱かれると、\nとても幸せな気分になるでしょう？\nふふ、実際はただエナジーを奪い取られようとしているだけなのに❤,Mmchu...❤ Being kissed and stroked at the same time just makes you feel so loved; doesn't it? Fufu; when in reality you're about to have your energy stolen~❤

> 唇が触れ合うだけで、脳が恋人と勘違いしちゃう❤\nチンポ触られただけで、好きって感情があふれてきちゃう❤\n人間ってほんと単純❤んっ、ちゅっ❤しこ、しこ・・・❤,Something as simple as our lips touching makes your mind mistake me for a lover.❤ Just having your cock touched makes feelings of romance flood through you.❤ Humans really are so simple.❤ Mm; chu~❤ Stroke; stroke...❤

Reviewer: ```suggestion
唇が触れ合うだけで、脳が恋人と勘違いしちゃう❤\nチンポ触られただけで、好きって感情があふれてきちゃう❤\n人間ってほんと単純❤んっ、ちゅっ❤しこ、しこ・・・❤,Something as simple as our lips touching causes your mind mistake me for a lover.❤ Just having your cock touched makes feelings of romance flood through you.❤ Humans really are so simple.❤ Mm; chu~❤ Stroke; stroke...❤
```

あら、わたしに気持ちよくしてほしいの？\nふふ、自分からエナジー盗んでくださいって頼んじゃうなんて、\n悪い子ね❤,Oh my; do you want me to make you feel good? Fufu; asking me to steal your energy all by yourself... You're such a bad boy~❤

> 悪いサキュバスとのバトルで、\n他の人間たち命もかかっていて、\nそんな場面でもその下半身のうずきを我慢できなかったのでしょう？,You're battling with an evil succubus; with other humans' lives at stake. Even in a situation like this; you couldn't resist the throbbing in your lower body; could you?

Reviewer: Hmm "couldn't resist the throbbing" is a bit of a weird English phrase.
```suggestion
悪いサキュバスとのバトルで、\n他の人間たち命もかかっていて、\nそんな場面でもその下半身のうずきを我慢できなかったのでしょう？,You're battling with an evil succubus; with other humans' lives at stake. Even in a situation like this; you couldn't refuse the call of your cock; could you?
```

> くすくす❤いいわ❤\nその浅ましい欲望に付き合ってあげる❤\nわたしに責められてどれぐらいで出してしまかしらね❤,*Giggle*❤ Alright.❤ I'll indulge that shameless desire of yours~❤ I wonder... Just how quickly will you cum when I start teasing you?❤

Reviewer: ```suggestion
くすくす❤いいわ❤\nその浅ましい欲望に付き合ってあげる❤\nわたしに責められてどれぐらいで出してしまかしらね❤,*Giggle*❤ Alright.❤ I'll indulge that shameless desire of yours~❤ I wonder... Just how quickly will you cum once I start teasing you?❤
```

うふふ❤手袋にこんなにかけて、\nわたしの手さばきですぐ骨抜きになってしまったわね❤,Ufufu~❤ You shot so much out onto my glove. My handiwork must've put you completely under my spell~❤
あん❤谷間が熱い❤\nどろどろの濃いエナジーが溶けだしてるわ❤\nあなたの裏切りのおかげで体力も全回復ね❤,Ahn~❤ My cleavage feels so hot!❤ Your thick; gooey energy is spilling out all over my breasts~❤ Thanks to your betrayal; my health is fully recovered.❤

> くす❤やはり男はこうやって胸に顔埋めると、\n我慢できずに漏らすように射精してしまうのね❤,*Giggle*❤ Just like I thought. When men have their faces buried in my chest like this; they can't help but cum like a fountain~❤

Reviewer: 埋める means the man buried his face. If she buried his face it would be  埋められる I believe.
漏ら means "leak". It's also associated with wetting themselves. Something like this might match better:
```suggestion
くす❤やはり男はこうやって胸に顔埋めると、\n我慢できずに漏らすように射精してしまうのね❤,*Giggle*❤ As expected; when a man buries their face between my tits; they can't help be leak everything out~❤
```

Author: I'm sorry, I personally refuse to use 'wetting themselves' as a euphemism for cumming. I can change it to leak, but the other way is just too juvenile and doesnt have the right associations in English

Reviewer: Changing to leak is fine by me!

Author: Okay, got about 'can't help be leak everything out'?

Reviewer: Yep that sounds good!

> 大事な命をこんなに簡単に盗まれて、\nずいぶんと脆弱なセキュリティしているわ❤\nおっぱいさえ当てていればいいんですもの❤くすくす❤,You let your precious life get stolen so easily. You have some real holes in your security; you know.❤ All I needed to do was press my breasts against you~❤ *Giggle*❤

Reviewer: Hmm I think the first sentence only makes sense if it's joined like the original.
```suggestion
大事な命をこんなに簡単に盗まれて、\nずいぶんと脆弱なセキュリティしているわ❤\nおっぱいさえ当てていればいいんですもの❤くすくす❤,To have your own precious lifeforce stolen so easily... You have some real holes in your security~❤ All I needed to do was press my breasts against you~❤ *Giggle*❤
```

Author: I don't think the 'and' makes sense here

Reviewer: The and is optional. Feel free to remove it if you like. If I imagine it as a speaking line the and makes sense but you're more than welcome to remove it if you'd like.

> あらあら、足裏をこんなに汚して、\nやっぱりマゾは踏んであげるのに限るわね。\n雑に刺激してあげるだけで一番簡単に射精させられるわ❤,Oh my; you made such a mess on my shoes. Just as I thought. Masochists like you are happiest when stepped on. A little rough stimulation is the easiest way to make you ejaculate~❤

Reviewer: 足裏 -> sole of the foot
あらあら -> oh dear (more of a condescending, shaking head kind of vibe). It's different to あら
  
 
```suggestion
あらあら、足裏をこんなに汚して、\nやっぱりマゾは踏んであげるのに限るわね。\n雑に刺激してあげるだけで一番簡単に射精させられるわ❤,Oh dear; you got the sole of my foot all dirty. Just as I thought; there's nothing quite like stepping on a masochist. A little rough stimulation is the easiest way to make them ejaculate~❤
```

あっ、ああっ・・・❤,Ahh; ahhh...❤

> あら、モンスターがわたしを見ながらオナニーをしてるわ❤\nくす、マスターばっかり狙って攻撃するから、\nうらやましくなってしまったのかしら❤,Oh my; one of my opponent's monsters is staring at me and touching himself.❤ *Giggle* Since I keep targeting and teasing your master; you must've gotten jealous.❤

Reviewer: Hmm the speaker appears to be talking to two different speakers in each of these sentences. They should be consistent otherwise it won't flow well.

```suggestion
あら、モンスターがわたしを見ながらオナニーをしてるわ❤\nくす、マスターばっかり狙って攻撃するから、\nうらやましくなってしまったのかしら❤,Oh my; one of my opponent's monsters is staring at me and touching himself.❤ *Giggle* Since I keep targeting and teasing his master; he must've gotten jealous.❤
```

Author: No, I think mine was right. She's ONLY addressing the monster here. It's as a result of her direct attacks so the point is she's not teasing the monster at all but he's still masturbating. The way you wrote it makes it sound like she's teasing both.

Reviewer: Hmm I can get completely behind that, but your first sentence isn't addressing the monster. It kind of sounds like she's talking to someone who _isn't_ the monster. As if she's making an annoucement. If that makes sense?

Author: I mean it's a simple third person rumination. I feel kind they're pretty common in general as well as in this game. I can see concern mixing them though. I can take a look.

Reviewer: Hmm. She does use 'かしら' in this sentence, which implies she's still ruminating, and NOT addressing anyone in particular. This would justify using my suggestion as well. I think?

Author: I mean if we cut the 'targeting him, ' i suppose it could work. The problem with that part is that she specifically hasn't been targeting him and its driving him crazy. So maybe either 'Since I keep ignoring him and teasing his master' or just 'Since I keep teasing his master'

Reviewer: Hmm I'm not really sure I understand what the flow issue is here. I think the sentence suggestion explains the nuance well. What's the problem exactly?

Reviewer: Discussed on discord and updated suggestion ^^

あなたもわたしにエナジーを盗んでほしいの？\nそれなら、ほら・・・❤,Do you want me to steal your energy; too? Then; come here...❤
わたしのキスをほっぺにプレゼントしてあげる❤\nんっ・・・ちゅっ❤,I'll give you a little present—a kiss on the cheek.❤ Mm...chu~❤
はひぃっ・・・❤,Hahhh...❤

> くす、ほっぺにキスされただけでイっちゃった❤\nキスマークをべっとり付けられたのが、\nよほど気持ちよかったみたいね❤,*Giggle* You came from just a kiss on the cheek.❤ Getting a lipstick mark plastered on your cheek must have felt really good~❤

Reviewer: ```suggestion
くす、ほっぺにキスされただけでイっちゃった❤\nキスマークをべっとり付けられたのが、\nよほど気持ちよかったみたいね❤,*Giggle* You came from just a kiss on the cheek.❤ Having a lipstick mark plastered on your cheek must have felt really good~❤
```

> こんなに簡単だと盗み甲斐もないわ❤\nもう少し理性を強くして、心の扉をきっちり閉めてくれた方が、\nお姉さん楽しいかも。くすくす❤,If it's this easy; there's really no challenge in stealing.❤ It'd probably be more fun for me if you strengthened your resolve a bit more and kept the doors to your heart closed a little tighter. *Giggle*❤

Reviewer: It's not so much about the challenge, but that she's saying there's no point in being sneaky if things are this easy:
```suggestion
こんなに簡単だと盗み甲斐もないわ❤\nもう少し理性を強くして、心の扉をきっちり閉めてくれた方が、\nお姉さん楽しいかも。くすくす❤,If things are this easy; there's not even any point to resort to espionage.❤ At least if you strengthened your resolve a bit more and kept the doors to your heart closed a little tighter; this would be much more fun. *Giggle*❤
```

Author: I'll take your note and have anther look. Your suggestion just doesnt flow well in my opinion

Reviewer: Alright. See what you can come up with ^^

Author: I added a subject. What do you think of this?

"A prize is hardly worth stealing if it's going to be this easy."

Reviewer: I like that!

> あら、盗みに行くまでもなく、\nマスターくんの方からエナジーを差し出してくれるの？\nうふふ❤いつの間にかあなたの心を盗み終わっていたみたい❤,Oh my; it seems there's no need to steel anything—the master is offering me his energy all on his own. Ufufu~❤ I guess that means I've already stolen your heart.❤

Reviewer: ```suggestion
あら、盗みに行くまでもなく、\nマスターくんの方からエナジーを差し出してくれるの？\nうふふ❤いつの間にかあなたの心を盗み終わっていたみたい❤,Oh my; it seems there's no need to steal anything—the cardmaster is offering me his energy all on his own. Ufufu~❤ I guess that means I've already stolen your heart.❤
```

わたしを熱っぽい目で見て「好き、好き」ってうわごとを言いながら、\nチンポをしこしこ❤しこしこ❤\n敵のことを好きになってしまうなんて悪い子ね❤,You're looking at me with so much passion in your eyes and; muttering "I love you" over and over while stroking your cock.❤ Stroke; stroke~❤ Falling in love with your enemy... You're such a bad boy~❤
わたしの虜になってしまったマスターくんに、\n気持ちいいご褒美よ❤\n濃厚なキスで何もかも吸い取ってあげる❤,Well then; here's a pleasurable reward for the cute little master who's already become completely captivated by me.❤ I'll drain everything from you with one. Deep. Kiss~❤
エナジーだけじゃなくて、魂ごとわたしに吸われてしまいなさい❤\nんっ・・・ちゅっ❤,But I want more than just your energy. Allow me to suck out your very soul.❤ Mmm... Chuu~❤
あひっ、ああっ・・・❤❤,Ahh; aahhh...❤❤
ふふ❤キスして少し舌を入れただけなのに、もうおしまいだなんて。\nそんなにビクビク大量に射精したら、あっという間に干からびちゃうわよ？,Fufu~❤ All I did was kiss you;—slip my tongue in just a little—and you finished already? If you cum so so much so vigorously; aren't you worried you'll shrivel away in no time?

> せっかく死ぬのならもう少し我慢すればいいのに。\n好きなお姉さんにねっとりキスされたら、\n男の子じゃ耐えられないか❤,If you're going to die anyway; you really should have held out a bit longer. I suppose when a boy gets kissed so passionately by the person he loves the most he just can't help it; can he?❤

Reviewer: ```suggestion
せっかく死ぬのならもう少し我慢すればいいのに。\n好きなお姉さんにねっとりキスされたら、\n男の子じゃ耐えられないか❤,If you're going to die anyway; you really should have held out a bit longer. But I suppose when a boy gets kissed so passionately by the person he loves the most he just can't help it; can he?❤
```

> ほら餞別代りにもう一度キスをしてあげる❤\nわたしの唇、死ぬ前に味わいなさい❤んっちゅ❤,Oh well. As a farewell gift; I'll give you one more kiss.❤ One last taste of my lips before you die.❤ Mmmchu~❤

Reviewer: Hmm maybe die could be a strong word?
```suggestion
ほら餞別代りにもう一度キスをしてあげる❤\nわたしの唇、死ぬ前に味わいなさい❤んっちゅ❤,Oh well. As a farewell gift; I'll give you one more kiss.❤ One last taste of my lips before you pass out.❤ Mmmchu~❤
```

Author: I mean, she's a succubus. He will die, you know?

Reviewer: Hmm OK I will resolve.

> あら、あなたも自分からわたしに盗まれに来ちゃうの？\nわたし人から与えられるより自分で盗みに行く方が好きなのだけど❤,Oh my; are you coming to let yourself be spirited away? I really prefer stealing things myself rather than having them given to me; you know.❤

Reviewer: 盗まれ means to be stolen/robbed, not spirited away.
```suggestion
あら、あなたも自分からわたしに盗まれに来ちゃうの？\nわたし人から与えられるより自分で盗みに行く方が好きなのだけど❤,Oh my; you're coming willingly to me to me take things from you? I really prefer stealing things myself rather than having them given to me; you know.❤
```

Author: I'll take a look later, your suggestion is really rough

Reviewer: Yeah it is a bit rough....
```suggestion
あら、あなたも自分からわたしに盗まれに来ちゃうの？\nわたし人から与えられるより自分で盗みに行く方が好きなのだけど❤,Oh my, you're coming to let me rob you of your own accord? I really prefer stealing things myself rather than having them given to me; you know.❤
```
I think I meant to phrase it like this?

Author: How about:

"Oh my, you're coming to let me rob you/steal from you of your own accord?"

Reviewer: If you think that flows better go for it ^^

> あら、あなたも自分からわたしに盗まれに来ちゃうの？\nわたし人から与えられるより自分で盗みに行く方が好きなのだけど❤,Oh my, you're coming to let me rob you of your own accord? I really prefer stealing things myself rather than having them given to me; you know.❤

Author: ```suggestion
あら、あなたも自分からわたしに盗まれに来ちゃうの？\nわたし人から与えられるより自分で盗みに行く方が好きなのだけど❤,Oh my; you're coming to let me rob you of your own accord? I really prefer stealing things myself rather than having them given to me; you know.❤
```

まあいいわ❤あなたのエナジーもサクっと奪いとってあげる❤\nしこしこ一生懸命しごいてるあなたのほっぺに・・・❤,Though; I suppose this is fine; too.❤ I'll quickly steal your energy as well~❤ I'll just place a kiss on your cheek while you're stroking yourself so desperatly...❤

> 怪盗からのキスマーク❤\nべっとりつけられて、その命ごとわたしのものになりなさい❤\nんっ・・・ちゅ❤,A gentle mark from the phantom thief.❤ Let it wash over you; and allow your very life to become mine.❤ Mmm... Chu~❤

Reviewer: ```suggestion
怪盗からのキスマーク❤\nべっとりつけられて、その命ごとわたしのものになりなさい❤\nんっ・・・ちゅ❤,A gentle kiss mark from the phantom thief.❤ Let it wash over you; and allow your very life to become mine.❤ Mmm... Chu~❤
```

Author: I don't think kiss is needed here. It's stated it's a kiss in the previous line, and I feel like the important thing is that he's been basically branded in a way, so specifying what kind of mark isn't really necessary.

> くす、ほっぺにキスされただけであなたも陥落ね❤\nもう少し我慢して、抵抗してくれた方が堕とし甲斐もあるのに❤\nチョロい男の子ばっかりで残念だわ❤,*Giggle* Just a kiss on the cheek and you've already fallen deeply for me; haven't you?❤ It would be more satisfying if you had resisted a bit longer.❤ All you boys are so easy; it's kind of disappointing~❤

Reviewer: Hmm that first line is quite different from the original Japanese. Here's what it should be:
```suggestion
くす、ほっぺにキスされただけであなたも陥落ね❤\nもう少し我慢して、抵抗してくれた方が堕とし甲斐もあるのに❤\nチョロい男の子ばっかりで残念だわ❤,*Giggle* I can see I've convinced you too; and all it took was just a kiss on the cheek~❤ Though it would be more satisfying if you had resisted a bit longer...❤ All you boys are so easy; it's kind of disappointing~❤
```

> エナジーはありがたくもらっておくけど、\n他の子たちはもう少し頑張ってほしいわね❤\nくすくす♪,While I'll take your energy gratefully; I really hope the other boys will try a little bit harder~❤ *Giggle*♪

Reviewer: "Take your energy gratefully" is a bit of a stilted English phrase we should try to avoid.
Maybe something like this:
```suggestion
エナジーはありがたくもらっておくけど、\n他の子たちはもう少し頑張ってほしいわね❤\nくすくす♪,While I'll gratefully accept the energy you're giving me; I really do hope the other boys will try a little bit harder~❤ *Giggle*♪
```

> せっかく死ぬのならもう少し我慢すればいいのに。\nんふふ、好きなお姉さんにねっとりキスされたら、\n男の子じゃ耐えられないか❤,If you're going to die anyway; you really should have held out a bit longer. Nfufu; I suppose when a boy gets kissed so passionately by the person he loves the most he just can't help it; can he?❤

Reviewer: ```suggestion
せっかく死ぬのならもう少し我慢すればいいのに。\nんふふ、好きなお姉さんにねっとりキスされたら、\n男の子じゃ耐えられないか❤,If you're going to die anyway; you really should have held out a bit longer. Nfufu; but I suppose when a boy gets kissed so passionately by the person he loves the most he just can't help it; can he?❤
```

Author: I really think die should be here. This is a life or death final boss and I feel the words should carry that weight.

Reviewer: Happy to leave that in. What about the addition of  'but'?

> うそ、驚いた。わたしのいる場所に的確に攻撃を当ててくるだなんて。\nあなた何か特殊な能力でも持っているのかしら。,Wow; I'm impressed. To think you could manage to land your hits on me so accurately. Do you maybe have some kind of special ability?

Reviewer: ```suggestion
うそ、驚いた。わたしのいる場所に的確に攻撃を当ててくるだなんて。\nあなた何か特殊な能力でも持っているのかしら。,Wow; I'm impressed. To think you could manage to land your hits on me so accurately. Perhaps you have some kind of special skill...?
```

> たまにいるのよね、異様に勘が鋭いというか、\nわたしの変装や潜伏が通用しない人。\nでもそういう相手に追い詰められたとき取る手は決まっているの❤,I run into people like that from time to time. Ones with unusually sharp instincts; people my disguises and stealth just don't seem to work on. But when cornered by such opponents; I have a special move I always use~❤

Reviewer: Just the flow is a bit stilted.
```suggestion
たまにいるのよね、異様に勘が鋭いというか、\nわたしの変装や潜伏が通用しない人。\nでもそういう相手に追い詰められたとき取る手は決まっているの❤,I run into people like that from time to time. Ones with unusually sharp instincts; people my disguises and stealth just don't seem to work on. But whenever I'm cornered by an opponent like that; there's a special move I have for just the occasion~❤
```

わたしのことが見えるのなら、ほら・・・❤\nしっかり見せつけてあげる❤,If you can see me; then look at this...❤ I'll give you good view of everything~❤
くす❤ジッパーを下ろしてあげたら、胸に目が吸い寄せられてる❤\n目は口ほどに物を言うとはいうけれど、\nあなたはずいぶん分かりやすいわね❤,*Giggle*❤ When I pull my zipper down like this; your eyes get drawn right to my breasts.❤ They say a person says more with their eyes than their lips; and you're quite easy to read; aren't you?❤

> この胸を使ってあなたの心を盗んであげるわ❤\nレザースーツで蒸れたわたしの谷間、\nサキュバスの濃厚なフェロモンでくらくらするほどいい匂いなのよ❤,I'll use these breasts to steal your heart~❤ My cleavage gets so steamy from my leather suit; and has such an intoxicating smell thanks to my thick succubus pheromones!❤

Reviewer: ```suggestion
この胸を使ってあなたの心を盗んであげるわ❤\nレザースーツで蒸れたわたしの谷間、\nサキュバスの濃厚なフェロモンでくらくらするほどいい匂いなのよ❤,I'll use these breasts to steal your heart~❤ My cleavage gets so steamy from my leather suit; and has such an intoxicating smell thanks to my thick succubus pheromones.❤
```

ねえ、わたしを捕まえるよりも、わたしに捕まえられたいでしょ？\nこの胸の間にむぎゅって拘束して、\n快楽で脳をおかしくしてあげる❤,Hey; rather than catching me; wouldn't you rather let me catch you? I'll squeeze you tightly between my breasts and drive your mind wild with pleasure~❤
理性を振り絞って攻撃した,Muster your reason and attack

> ふらふらと胸に吸い寄せらえてしまった,Agh… Her boobs... I... I can't...!'

Reviewer: ```suggestion
ふらふらと胸に吸い寄せらえてしまった,'I... I can't stop looking at her soft breasts...'
```

あうっ！くっ、男のくせになんて意志の強さなの…。,Ahh! Guh; for a man to have such incredible willpower...
あら、おっぱいのことが大好きなくせに頑張るのね❤\nでも、動揺して攻撃が雑になってる。\nさっきまでの鋭い攻撃が見る影もないわ。,Oh my; despite your love for breasts; you're still trying so hard; aren't you?❤ But now you're flustered; and your attacks have gotten sloppy. Those sharp strikes from before are nowhere to be seen.
あなた女の胸が好きなんでしょう？\nうふふ、全然隠せてないわよ❤そんなおっぱいフェチの男が\nサキュバスの谷間を前に理性を保てるわけないじゃない。,You can't resist a pair of breasts; can you? Ufufu; you can't hide it at all!❤ There's no way a tit-obsessed man like you could maintain his reason in front of a succubus's cleavage.
これでどうかしら❤\n胸を手で持って、ゆっくりと揺らしてあげる❤,How about this?❤ I'll hold my breasts with my hands and slowly shake them for you~❤
う、ううっ❤やめて、あううっ・・・❤,Oh; ugh.❤ Stop; ahhh...❤
くすくす❤もう限界？我慢できない？\n自分の欲望に嘘をつくのはよくないわ。\nあなたのしたいことに、素直になりなさい❤,*Giggle*❤ Already at your limit? Can't hold back anymore? It's not good to lie about your own desires; you know. You should really be more honest about what you want~❤

> 欲しいものには何でも手を伸ばす、それが怪盗❤\n,The will to reach for whatever I want—that's what makes me a phantom thief.❤

Reviewer: I believe the Japanese is more saying "that is the phantom thief way"
```suggestion
欲しいものには何でも手を伸ばす、それが怪盗❤\n,The will to reach for whatever I want—that is the way of the phantome thief.❤
```

> 欲しいものには何でも手を伸ばす、それが怪盗❤\n,The will to reach for whatever I want—that is the way of the phantome thief.❤

Reviewer: phantome 

```suggestion
欲しいものには何でも手を伸ばす、それが怪盗❤\n,The will to reach for whatever I want—that is the way of the phantom thief.❤
```

わたしの蒸れた谷間に顔をうずめて、\nぱふぱふ❤むぎゅう❤\nやわらかいおっぱいで快楽を味合わせてあげる❤,So I'll bury your face in my steamy cleavage and... Puff-puff~❤ Squeeze~❤ I'll use my breasts to give you a taste of true pleasure!❤
余計なプライドなんて捨てて、\nお姉さんのものになりなってしまいなさい❤,Throw away any unnecessary pride you still cling to and become mine completely~❤

> 理性を振り絞り一気に前進して攻撃した,Muster your reason and rushed forward

Reviewer: ```suggestion
理性を振り絞り一気に前進して攻撃した,Muster your reason and rush forwards
```

> 揺れるおっぱいで何も考えられなくなってしまった,'Their bounce... I... I can't...'

Reviewer: ```suggestion
揺れるおっぱいで何も考えられなくなってしまった,'The way they bounce... I... I can't...
```

> 揺れるおっぱいで何も考えられなくなってしまった,'Their bounce... I... I can't...'

Reviewer: ```suggestion
揺れるおっぱいで何も考えられなくなってしまった,'The way they bounce... I... I can't... think...'
```

きゃあっ！\nううっ、こんなやみくもな攻撃にやられるだなんて。悔しい・・・。,Kyaa! Ugh; to think I'd be hit by such a reckless attack. How humiliating...
はい、捕まえた❤\nくすくす❤棒立ちでおっぱいに見惚れて、かわいいわね❤\nさっきまであんなに強かった男とは思えないわ❤,There; caught you~❤ *Giggle*❤ You were just standing there mesmerized by my breasts. You're rather cute; aren't you?❤ I can't believe you're the same man who nearly ended me just moments ago.❤

> ほら、むぎゅうって抱き寄せながら、\nぱふぱふ❤ぱふぱふ❤\nどう？サキュバスの谷間は気持ちいい？,Here; I'll pull you even closer. Puff-puff~❤ Puff-puff~❤ How does it feel? Isn't the cleavage of a succubus wonderful?

Reviewer: 'Isn't the cleavage of a succubus wonderful?' feels a little weird to me. Maybe something like this?

```suggestion
ほら、むぎゅうって抱き寄せながら、\nぱふぱふ❤ぱふぱふ❤\nどう？サキュバスの谷間は気持ちいい？,Here; I'll pull you even closer. Puff-puff~❤ Puff-puff~❤ Well? Succubus breasts feel good; don't they?
```

Author: I don't know, I feel like it should still be a comment on how nice her breasts are. Next time I'm at my laptop I'll take another look

Reviewer: It is a comment on how nice it is though. You could get more explicit by saying "the feeling of being between a pair of Succubus breasts"?

Author: I just feel like the 'being' part takes away from it. It's a comment on the quality of her breasts, but the experience

Reviewer: But it is a comment on the quality because the Japanese is saying her cleavage feels good.

そのうえ誘惑に負けてしまったあなたは\n心のバリアも解けている状態❤\n脳がどろどろになるぐらい気持ちいいはずよ❤,On top of that; since you gave in to temptation; your mental barriers are down; too.❤ It should feel so good it turns your brain to mush!❤
わたしの甘いフェロモンが、鼻と口から\n全身に入り込んで、体中を快感で焼いていく❤\n悶えるように熱くなって、ほらチンポもこんなに反応してる❤,My sweet pheromones enter through your nose and mouth; spreading throughout your whole body; burning you with pleasure from within.❤ You're getting feverishly hot as you writhe; and look; even your cock's reacting so strongly~❤
もう心は陥落しちゃったようだし、\n次はお姉さんにエナジーを盗まれたいのよね❤\n胸に顔をうずめさせながらほら、両手でしこしこしこ♪,It looks like your heart has already fallen for me; so now you want me to steal all your energy; don't you?❤ I'll let you bury your face in my breasts; so relax and let me stroke you with both hands~♪
はひっ、あっ、あっ、気持ちいいっ❤❤ああっ❤,Hahhh; ah; ah; it feels so good.❤❤ Aahh!❤
うふふ、いい喘ぎ声❤\nお姉さんの手のテクニックはどうかしら？\nあらゆる鍵を破ってきただけあって手先の器用さは抜群よ❤,Ufufu; such lovely little moans~❤ How do you like my hand technique? Having practiced to break all kinds of locks; my dexterity is exceptional!❤

> 革の手袋の摩擦が気持ちいいでしょう？\n少し激しい刺激だけれど、\nその分、顔は優しい快楽に甘やかされてちょうどいいわね❤,The friction from my leather gloves feels good; doesn't it? The stimulation might be a bit intense; but with your face being pampered with such gentle pleasure; it balances out perfectly; don't you think?❤

Reviewer: The phrase just feels a little off to me, maybe something like this?
```suggestion
革の手袋の摩擦が気持ちいいでしょう？\n少し激しい刺激だけれど、\nその分、顔は優しい快楽に甘やかされてちょうどいいわね❤,The friction from my leather gloves feels good; doesn't it? The stimulation might be a bit intense; but with your face being gently pampered so pleasurably; it balances out perfectly; don't you think?❤
```

Author: I don't think that works. The pleasure is what is gentle. It's a quality of the feeling, but the pampering.

Reviewer: Hmm yeah I don't know what I was thinking with this. I'll resolve.

> しこしこ、しこしこってされて\n気持ちよすぎて悶えてしまったら、\nぎゅうって顔をおっぱいに押しつけていいの❤,If being stroked like this makes you feel so good you writhe in pleasure; you're welcome to press your face even tighter against my breasts~❤

Reviewer: "even deeper' is probably a more natural phrase to use here!
```suggestion
しこしこ、しこしこってされて\n気持ちよすぎて悶えてしまったら、\nぎゅうって顔をおっぱいに押しつけていいの❤,If being stroked like this makes you feel so good you writhe in pleasure; you're welcome to press your face even deeper into my breasts~❤
```

> フェロモンのせいで余計我慢できなくなるし、\n快感も倍増しちゃうけれど、\nくすくす❤一瞬でも柔らかい感触に甘えられるなら男は満足よね❤,The pheromones will make you even less able to hold back; and the pleasure will double. Oh; but be careful. *Giggle* If a man indulges in such soft pleasure for even a moment; he's bound to surrender; wouldn't you think?❤

Reviewer: 'If a man indulges in such soft pleasure' is a bit weird for an English phrase. Maybe try something like this:

```suggestion
フェロモンのせいで余計我慢できなくなるし、\n快感も倍増しちゃうけれど、\nくすくす❤一瞬でも柔らかい感触に甘えられるなら男は満足よね❤,The pheromones will make you even less able to hold back; and the pleasure will double. Oh; but be careful. *Giggle* If a man indulges in things that are soft and pleasurable for even a moment; he's bound to surrender; wouldn't you think?❤
```

Author: How about something like:

"If you offer a man even a moment of gentle pleasure"

Reviewer: I don't think the phrase "gentle pleasure" isn't a common phrase though. You can say "gently pleasure", but "gentle pleasure" is a bit stilted.

Author: I really feel like it is common though. Different kinds of pleasure are very real. It is soft pleasure as opposed to rough pleasure, which i know for a fact is a phrase we've used in this translation.

> わたしもだいぶダメージを受けてそろそろ限界だし、\nほら、すぐに屈服していいわよ❤,I've taken quite a bit of damage myself; and I'm bound to reach my own limit soon. So go on; hurry up and submit to my touch!❤

Reviewer: Another case of いいわよ, which is more that she's saying 'it's okay/consenting' it's not really a command. Maybe something like this?

```suggestion
わたしもだいぶダメージを受けてそろそろ限界だし、\nほら、すぐに屈服していいわよ❤,I've taken quite a bit of damage myself; and I'm bound to reach my own limit soon. So go on; hurry up and submit; there's no need to worry~❤
```

Author: How about "It's okay, there's no need to hold back/resist"

Reviewer: Hmm I think my original suggestion flows better! She's downed at this point, so it doesn't make sense for her to tell him to resist.

Author: It just feels tacked on to the end and unnatural. Like an after thought

Reviewer: It's just a subtle way that captures the nuance of いいわよ (its ok). It's not an afterthought, its an intentional way of capturing the Japanese meaning while flowing well.

It's just stuff like that you have to look out for ^^

うふ❤わたしの手技に、男が耐えられるわけないわよね❤\nこんなに手袋をどろどろにして、\n男の子からエナジーを盗むのってやっぱり簡単だわ❤,Ufu~❤ No man could possibly resist my handjob; could he?❤ Making my opponents make a mess of my glove while stealing energy from boys is always so easy~❤
あらあら、わたしにエナジー貢ぎたくて、\nさっそく屈服しちゃったようね❤\n踏まれて貢いで喜んで、マゾの男ってかわいい❤くすくす♪,Oh my; it seems you were so desperate to offer me your energy you've already submitted; haven't you?❤ Being stepped on while offering me tribute so happliy... Masochistic men are rather adorable~❤ *Giggle*♪
{自分}に敗れた僕は、薄暗い地下通路でエナジーを失い、地面に膝をついていた。,After losing to {自分} and having my energy drained; I fell to my knees on the ground in the dim underground passage.
動けなくなった僕を横目にエルルは他のマスターたちのところに向かっていく。下級のマスターたちではきっと歯が立たない。このままではみんなやられてしまう・・・。,Stuck where I was; unable to move; I watched as Eruru headed off toward the other masters. The lower-ranked cardmasters definitely wouldn't stand a chance. At this rate; everyone would be defeated...

> くす、Bランク最強のマスターのくせにあっけないものね❤\nわたしはエルルをサキュバスにした、その張本人。\nそんな悪い女にメロメロにされちゃって、心が痛まないのかしら❤,*Giggle* For the 'strongest B-rank'; you're actually pretty pathetic; aren't you?❤ I'm the very one who turned Eruru into a succubus; you know. Doesn't your heart ache knowing you've fallen-head-over heels for such a wicked woman?❤

Reviewer: ```suggestion
くす、Bランク最強のマスターのくせにあっけないものね❤\nわたしはエルルをサキュバスにした、その張本人。\nそんな悪い女にメロメロにされちゃって、心が痛まないのかしら❤,*Giggle* For the 'strongest B-rank cardmaster'; you're actually pretty pathetic; aren't you?❤ I'm the very one who turned Eruru into a succubus; you know. Doesn't your heart ache knowing you've fallen-head-over heels for such a wicked woman?❤
```

> 快楽を与えられれば、\n大事な仲間を奪った相手にすらそんな蕩けた顔を向けてしまう❤\nふふ、人間って本当に愚かな生き物ね❤,When given pleasure; you show such a melted expression; even to the one who stole your precious companion from you.❤ Fufu; humans really are such foolish creatures; aren't they?❤

Reviewer: 'you show such a melted expression' is a bit weird English. Maybe something like this could be better?
```suggestion
快楽を与えられれば、\n大事な仲間を奪った相手にすらそんな蕩けた顔を向けてしまう❤\nふふ、人間って本当に愚かな生き物ね❤,You make such dopey expressions when you're pleasured; even to the one who stole your precious companion away.❤ Fufu; humans really are such foolish creatures; aren't they?❤
```

Author: I kinda like leaving 'stole' in there to lean into the 'thief' aspect. For the other part, what about:

"Your resistance melts the moment you're given pleasure"

Or something along those lines?

Reviewer: Hmm first line I think if you can fit stole in a way that flows naturally go for it.

Not sure what you mean by the other part. I don't see what Japanese line that's supposed to be?

Author: Just trying something out. It's using 'resistance' instead of 'face' as a localization choice since i think the melting part is important but you don't like the melting face idea

Reviewer: Hmm lets keep the dopey expression phrase. It's much more common and keeps the same Japanese nuance well, instead of us going for a completely different meaning when we don't need to.

あううっ・・・❤くそぉ・・・❤,Agh...❤ Damn it...❤
彼女の言う通り、エルルさんをサキュバス化させたこの怪盗サキュバスは、僕がいま一番負けてはいけない相手のはずだ。それなのに・・・。,She was right. This phantom thief succubus who had turned Eruru into a succubus was the one opponent I absolutely couldn't afford to lose to. And yet...

> 彼女に軽く触れられただけで心を奪われて、ほとんど抵抗もできないまま射精に追い込まれてしまった。いまも目の前にいる彼女の顔から目をそらすことができなくなっている。,Just from her light touch; my heart had been stolen away; and I was driven to ejaculation with barely any resistance. Even now; I couldn't bring myself to look away from her face while it was right in front of me.

Reviewer: I don't know if this is strictly better or not per say, but it could work:
```suggestion
彼女に軽く触れられただけで心を奪われて、ほとんど抵抗もできないまま射精に追い込まれてしまった。いまも目の前にいる彼女の顔から目をそらすことができなくなっている。,Just from her light touch; my heart had been stolen away; and I was driven to ejaculation with barely any resistance. Even now; I couldn't bring myself to look away from her face while she was right in front of me.
```

（このままじゃダメだ・・・❤足に力を込めて立ち上がらないと・・・❤）,(It can't end like this...❤ I need to put strength in my legs and stand up...❤)
だが、僕の体に力は入らない。激しい運動で疲れ果てたときのように体が重い。これがサキュバスに射精させられるということなのか。,But my body wouldn't respond. It felt so heavy; as if exhausted after intense exercise. Is this what it means to be made to ejaculate by a succubus?
あなたのエナジーはすでにわたしに大量に盗まれたのよ❤\nカードを使って戦うことなんて、とてもできないでしょう？,I've already stolen a massive amount of your energy!❤ You couldn't possibly use your cards to fight anymore; could you?
心の方は、まだ少しは抵抗できるみたいだけれど、\nそのとろんとした目を見る限り、\nどうせ時間の問題ね❤,Your mind seems like it can still resist a little; but looking at those glazed eyes; it's only a matter of time now; isn't it?❤
さあ、あなたのことを完全に堕としてあげる❤\n吸いつくすか、奴隷にするか、\nうふふ、どちらも楽しそう❤,Now; I'll corrupt you completely!❤ Should I drain you dry; or make you my slave? Ufufu; both sound so entertaining~❤

> あなたの弱点を、やさしく握って・・・❤\nしこ、しこ、しこ・・・❤,Either way; I'll start by grasping your weakness...❤ Stroke stroke stroke...❤

Reviewer: Hmm what about something like this? Would this be better?
```suggestion
あなたの弱点を、やさしく握って・・・❤\nしこ、しこ、しこ・・・❤,Either way; I'll start by lightly grapsing you're weakest spot...❤ Stroke stroke stroke...❤
```

Author: How about 'your weakest point/spot'?

Reviewer: Yep that also works!

うあっ❤あっ、ああっ❤指がからみついてきてっ❤,Ugh!❤ Ah! Aah!❤ Your fingers are coiling around me!❤

> わたしの指先、よく動くでしょう？\n鍵開けも、複雑な警報の解除もお手の物。\n泥棒にとって指先の器用さは命綱ですもの❤,My fingertips move quite well; don't they? Lock-picking and disabling complex alarms are my specialty. For a thief; nimble fingers are our most important tool~❤

Reviewer: optional
```suggestion
わたしの指先、よく動くでしょう？\n鍵開けも、複雑な警報の解除もお手の物。\n泥棒にとって指先の器用さは命綱ですもの❤,My fingertips move quite well; don't they? Lock-picking and disabling complex alarm systems are my specialty. For a thief; nimble fingers are our most important tool~❤
```

> もちろん、男の子の心をこじ開けてしまうのも得意よ❤\n五本の指を、それぞれ別の生き物みたいに、\nうねらせて、絡ませて・・・❤しこしこしこ❤,Of course; I'm also skilled at prying open boys' hearts.❤ Moving all five fingers; each with a mind of its own; shifting and writhing...❤ Stroke stroke stroke~❤

Reviewer: Would 'wriggling' be a better word? Or coiling/sliding?

```suggestion
もちろん、男の子の心をこじ開けてしまうのも得意よ❤\n五本の指を、それぞれ別の生き物みたいに、\nうねらせて、絡ませて・・・❤しこしこしこ❤,Of course; I'm also skilled at prying open boys' hearts.❤ Moving all five fingers; each with a mind of its own; shifting and sliding about...❤ Stroke stroke stroke~❤
```

Author: Went with 'coiling'
